suites/suites/suites.yaml

```diff
@@ -675,11 +675,6 @@ tests:
     tests:
       - cosmo_tester/test_suites/test_blueprints -e puppet -e chef -e docker -e windows
 
-  openstack_blueprints_persistence_recovery:
-    tests:
-      - cosmo_tester/test_suites/test_blueprints/docker_persistence_test.py
```

Comment:
make the commit message + PR message state that this change is only relevant for m5 and should not be merged to master (m6) as well